File: Iniciante/URI 1016.kt
import java.util.Scanner;

fun main(args: Array<String>) {
    val leitor = Scanner(System.`in`);
    
    val km: Int = leitor.nextInt();
    val minutos: Int = (60 * km) / 30;
    
    println(minutos + " minutos");
}

File: Iniciante/URI 1016.scala
import java.util.Scanner

object Main {

    def main(args: Array[String]) {
        val leitor = new Scanner(System.in)
        val km = leitor.nextInt()
        val minutos = (60 * km) / 30
        println(minutos + " minutos")
    }
	
}